packages/api/src/email-campaign/dto/email-campaign-input.factory.ts

```diff
@@ -27,13 +31,13 @@ export class EmailCampaignInputFactory {
             @Field()
             subject: string;
 
-            @IsUndefinable()
+            @IsNullable()
```

Comment:
```suggestion
      @IsUndefinable()
```

The type is not Date | null

Reply:
Adapted in [9ff247e](https://github.com/vivid-planet/comet-brevo-module/pull/32/commits/9ff247ecf83b670fe2fc5ee37a2b6ffbc63a50d8)